binaries/apiserver/main.go

```diff
@@ -25,7 +25,7 @@ func main() {
 	configFlag := flag.String("config", "{}", "API Server Config (either a filename like local.local or JSON text")
 	flag.Parse()
 
-	log.SetFlags(log.LstdFlags | log.LUTC | log.Lshortfile)
+	// log.SetFlags(log.LstdFlags | log.LUTC | log.Lshortfile)
```

Comment:
Should've said this in the summary, but I was planning to include file:Line# in the subsequent Twitter source phab, I think it makes sense to leave OSS as barebones as possible and do all custom configuration within source.